static/app/components/avatar/gravatar.tsx

```diff
@@ -26,24 +45,23 @@ function Gravatar({
   onLoad,
   suggested,
 }: Props) {
-  const isMountedRef = useIsMountedRef();
-  const [SHA256, setSHA256] = useState<typeof HasherHelper>();
-
-  const loadSHA256Helper = useCallback(async () => {
-    const mod = await import('crypto-js/sha256');
-
-    if (isMountedRef.current) {
-      // XXX: Use function invocation of `useState`s setter since the mod.default
-      // is a function itself.
-      setSHA256(() => mod.default);
+  const [sha256, setSha256] = useState<string | null>(null);
+  useEffect(() => {
+    if (!isCryptoSubtleDigestAvailable()) {
+      return;
     }
-  }, [isMountedRef]);
 
-  useEffect(() => {
-    loadSHA256Helper();
-  }, [loadSHA256Helper]);
+    hashGravatarId((gravatarId ?? '').trim())
+      .then(hash => {
+        setSha256(hash);
+      })
+      .catch(() => {
+        // If there is an error with the hash, we should not render the gravatar
+        setSha256(null);
```

Comment:
could also do nothing here 🤷

Reply:
@scttcper We should log failures to ensure we dont silently swallow errors